squiRL/common/agents.py

```diff
@@ -67,7 +67,7 @@ def get_action(
         action_logit = net(obs)
         probs = F.softmax(action_logit, dim=-1)
         dist = torch.distributions.Categorical(probs)
-        action = dist.sample().squeeze().numpy()
+        action = dist.sample().squeeze().cpu().numpy()
```

Comment:
does it need detach() as well?

Reply:
no it doesnt. I tested it with 1 gpu and 2 gpus. no `detach` needed.